xchange-blockchain/src/main/java/org/knowm/xchange/blockchain/params/BlockchainWithdrawalParams.java

```diff
@@ -0,0 +1,30 @@
+package org.knowm.xchange.blockchain.params;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.service.trade.params.WithdrawFundsParams;
+
+import java.math.BigDecimal;
+
+@Data
+@Builder(toBuilder = true)
+@Jacksonized
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BlockchainWithdrawalParams implements WithdrawFundsParams {
+
+    /**
+     * UUID that represent a whitelisted address
+     * to get the UUIDs of the whitelisted addresses, call this endpoint
+     * @see <a href="https://api.blockchain.info/v3/#/payments/getWhitelist">https://api.blockchain.info/v3/#/payments/getWhitelist</a>
```

Comment:
Maybe it would be worth to indicate that whitelist can be created through the exchange ui.

Reply:
Is it easier to understand if I put it this way?

/**
     * UUID representing a whitelist address that is indicated in the whitelistId field.
     * To obtain the UUIDs of the whitelist addresses, call this endpoint:
     * @see <a href="https://api.blockchain.info/v3/#/payments/getWhitelist">getWhitelist</a>
     */

Reply:
Put something like this: "Whitelist addresses can be managed in Blockchain Exchange UI"